extensions/arc/src/ui/dashboards/postgres/postgresComputeAndStoragePage.ts

```diff
@@ -155,12 +161,13 @@ export class PostgresComputeAndStoragePage extends DashboardPage {
 						},
 						(_progress, _token) => {
 							return this._azdataApi.azdata.arc.postgres.server.edit(
-								this._postgresModel.info.name, this.saveArgs);
+								this._postgresModel.info.name, this.saveArgs).then(
```

Comment:
This change was moved into a separate PR right?

Reply:
Yup, https://github.com/microsoft/azuredatastudio/pull/13254 .